install/build-docker-images.sh

```diff
@@ -1,6 +1,8 @@
 echo "${_group}Building and tagging Docker images ..."
 
 echo ""
+# Build sentry-self-hosted-local first as it is needed for sentry-cleanup-self-hosted-local
+$dc build --force-rm web
```

Comment:
Comment says "Build sentry-self-hosted-local first" but then we "build ... web"?